include/deal.II/matrix_free/tensor_product_kernels.h

```diff
@@ -296,14 +296,20 @@ namespace internal
      *             derivatives, 2 second derivates. Note that all the
      *             derivatives access the data in @p shape_values passed to
      *             the constructor of the class
+     * @tparam lex_faces Sets how the evaluation points on the faces should be
+     *                   sorted: lexicographically or deal.II-style (special
+     *                   treatment of orientation 1 in 3D). Per default
+     *                   deal.II-order is enabled. Note: that the deal.II
+     *                   ordering is only working for dimensions up to 3.
```

Comment:
I would not call it `deal.Il` orientation but rather `right-hand-system` numbering or something like that (which is a 3D concept). The idea is to let axis direction wrap around in defining the order, rather than going lexicographically from the faster running tensor index to the slower running one. What do you think?

Reply:
> What do you think?

I agree. I just did not know how to call it. I have made the change!